### PR TITLE
Sumaclust recipe

### PR DESCRIPTION
**Pull request by JTFouquier**
* [X ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X ] This PR adds a new recipe.
* [X ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by JTFouquier** [comment]
It said it was able to be merged, but now it's saying checks aren't completed yet. Happy to wait. :) Let me know if there's anything I can improve. I'm not really familiar with this process. Thanks!

**Comment by JTFouquier** [comment]
Hi there, sorry, I'm only able to see the "fail" when I submit a PR. It looked good before. Any thoughts? Thank you!

screen shot 2018-08-13 at 6 02 35 pm

**Comment by bgruening** [comment]
Mh, the error here is `sumaclust: command not found` :(

File: recipes/sumaclust/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  url: https://git.metabarcoding.org/obitools/sumaclust/uploads/59ff189079b9e318f07b9ff9d5fee54b/sumaclust_v1.0.31.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - sumaclust -h > /dev/null
```
Comment (bgruening) [inline recipes/sumaclust/meta.yaml]: You can remove the `> /dev/null`

File: recipes/sumaclust/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+about:
+  summary: Sumaclust. For more information see url.
```
Comment (bgruening) [inline recipes/sumaclust/meta.yaml]: A short summery would be nice.

File: recipes/sumaclust/conda_build.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+bash conda_build.sh
```
Comment (bgruening) [inline recipes/sumaclust/conda_build.sh]: conda_build.sh is not part of this PR.
I have checked the tarball and there is a makefile included. Does `make install` work instead?

Comment (JTFouquier) [reply recipes/sumaclust/conda_build.sh]: `make` by itself is what is used to install it locally.

File: recipes/sumaclust/meta.yaml (hunk 2 above)
Comment (JTFouquier) [reply recipes/sumaclust/meta.yaml]: They have the longest 'summary'. Didn't want to butcher it but will try to do it justice. :) Thanks.

File: recipes/sumaclust/conda_build.sh (hunk 3 above)
Comment (JTFouquier) [reply recipes/sumaclust/conda_build.sh]: Thanks for your help @bgruening!  I tried naming the file `build.sh` and it fails (wish some sort of hint to rename it as `conda_build.sh`, and when I do name it `conda_build.sh` it works for me locally. Everything is green before I change things. I attempted to add `make` into the bash script because that is how to install it locally. BTW I'm definitely more bioinformatics than CS if I sound confused. :grin: I have a few more tools I need to get into Bioconda, so any help is great so I can repeat the process. 

Thank you!

**Comment by bgruening** [comment]
Oh ok. But this one is normal. This is created internally. What you see here is that your tool is actually working unter Linux but crashing under OSX.
This is because the makefile wants to use openmp as you can see in the error message: `clang: error: unsupported option '-fopenmp'`. And this is because we try to build with clang under OSX and not GCC.

**Comment by bgruening** [comment]
@JTFouquier all green. My last commits are not really recommended its a hack to get your tool build on OSX. Alternatively, you could also patch your tool to be able to use LLVM (clang). Let the other tools coming :)

**Comment by JTFouquier** [comment]
Thanks so much for your help and for being so fast!!
